gradle/libs.versions.toml

```diff
@@ -11,7 +11,7 @@ jackson-protobuf = "0.9.13"
 kotlin = "1.7.22"
 kotlin-coroutines = "1.6.4"
 kotlin-logging = "3.0.4"
-logback = "1.4.5"
+logback = "1.2.3"
```

Comment:
Noticed this artifact wasn't actually behaving properly.  This downgrade gets tests that use it to work correctly with logging

Reply:
THEY BROKE LOGGING!?

Reply:
AHHHHHHH